.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "08:00"
+    timezone: US/Central
+  open-pull-requests-limit: 10
```

Comment:
@swalkinshaw Should we remove these lines to use GitHub default behaviours only?

Reply:
I'm fine either way... don't think it matters too much for a lower-volume repo like this one. If you want to remove after, feel free